### ISSUE TITLE
expandLineSelection: Default Hotkey Conflict

### ISSUE THREAD

**Issue by yxrkt** [Microsoft/vscode#71216]
Issue Type: <b>Bug</b>

expandLineSelection was changed recently from Ctrl+I to Ctrl+L. This is problematic because o.showOutput is bound to Ctrl+L L by default. When I hit Ctrl+L, Code waits for a second key chord.

VS Code version: Code 1.32.3 (a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4, 2019-03-14T23:43:35.476Z)
OS version: Windows_NT x64 10.0.15063

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-5960X CPU @ 3.00GHz (16 x 3000)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|63.84GB (51.59GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
TabOut|alb|0.1.6
folder-indexing|ast|0.2.0
insertnumbers|Asu|0.9.1
vscode-svgviewer|css|2.0.0
xml|Dot|2.4.0
vscode-npm-script|eg2|0.3.5
jsonnet|hep|0.1.0
Ionide-fsharp|Ion|3.34.0
reg|ion|0.0.2
cpptools|ms-|0.22.1
csharp|ms-|1.18.0
PowerShell|ms-|1.11.0
vscode-autohotkey|sle|0.2.2
t4-support|zbe|0.4.2

</details>

**Comment by alexandrudima** [comment]
@yxrkt That is not a default keybinding on Windows, i.e. by default Ctrl+L does not collide:

![image](https://user-images.githubusercontent.com/5047891/55107150-e102fd80-50d0-11e9-9bae-86759e585f33.png)

**Comment by siesing** [comment]
I'm experiencing the exact same issue as @yxrkt. Very, very annoying. One of my most used shortcuts.
![image](https://user-images.githubusercontent.com/4631265/55476511-16e73b00-5617-11e9-8c6f-16171c1867a9.png)

**Comment by alexandrudima** [comment]
@siesing All our keybindings are customizable. Feel free to rebind. Right click on `expandLineSelection` and choose `ctrl+i`.

**Comment by siesing** [comment]
@alexandrudima of course they are(since vscode is the best thing since sliced bread) and of course, I've changed it back to the "the old" ctrl+i. But no one will be able to do expandLineSelection with the default settings if they install the OmniSharp extension. Just wanted to point that out.

**Comment by alexandrudima** [comment]
@siesing Haa! Thanks for pointing out which extension it was. I have reached out to Omnisharp to kindly ask if they can remove their keybinding contribution in https://github.com/OmniSharp/omnisharp-vscode/issues/2962.

---

To summarise:

**TL;DR** Starting with 1.32, `expandLineSelection` is now bound to <kbd>Ctrl</kbd>+<kbd>L</kbd> and no longer to <kbd>Ctrl</kbd>+<kbd>I</kbd>. Out of the box, there is no conflict and it works under <kbd>Ctrl</kbd>+<kbd>L</kbd>.

Omnisharp binds show output to a chord which starts with <kbd>Ctrl</kbd>+<kbd>L</kbd>. I have reached out to them to remove that in https://github.com/OmniSharp/omnisharp-vscode/issues/2962

To unblock yourself, you can edit the keybindings and search for `expandLineSelection` and bind it back to <kbd>Ctrl</kbd>+<kbd>I</kbd>.
